workspaces/scanner/test/extractors/payload.spec.ts

```diff
@@ -179,6 +179,30 @@ describe("Extractors.Probes", () => {
     });
   });
 
+  describe("FlagsExtractor", () => {
+    it("should extract strnum flags", () => {
+      const extractor = new Extractors.Payload(
+        strnumNodesecurePayload,
+        [
+          new Extractors.Probes.FlagsExtractor()
+        ]
+      );
+
+      const {
+        flags
+      } = extractor.extractAndMerge();
+
+      assert.deepEqual(
+        flags,
+        {
+          hasWarnings: 1,
+          isOutdated: 1,
+          hasManyPublishers: 1
+        }
```

Comment:
We did have that before?

Reply:
Yes https://github.com/NodeSecure/scanner/blob/master/workspaces/scanner/test/fixtures/extractors/strnum.json#L18